README.md

```diff
@@ -750,6 +751,43 @@ task runtime_test {
 }
 ```
 
+## Specifying Default Values
+
+Default values for runtime attributes can be specified via [workflow options](#workflow-options).  For example, consider this WDL file:
+
+```wdl
+task first {
+  command { ... }
+}
+
+task second {
+  command {...}
+  runtime {
+    docker: "my_docker_image"
+  }
+}
+
+workflow w {
+  call first
+  call second
+}
+```
+
+If this workflow were to be run with the following workflow options:
+
+```json
+{
+  "defaultRuntimeOptions": {
+    "docker": "ubuntu:latest",
+    "zones": "us-central1-a us-central1-b"
+  }
+}
+```
+
+Then these values for `docker` and `zones` will be used for any task that does not explicitly override them in the WDL file.
+
+In the example above, `task first` will inherit from the `defaultRuntimeOptions` section of the workflow options.  `task second` will only use the "zones" information from the workflow options.
```

Comment:
Oh, I got confused. Maybe `task second will use zones from the default in workflow options but because docker is specified in the WDL, that value will be used`